app-editors/bluefish/bluefish-2.2.17.recipe

```diff
@@ -6,11 +6,13 @@ and markup languages. For an extensive overview, see \"Features\" on the website
 Bluefish is a multi-platform application that runs on most desktop operating systems, including \
 Linux, FreeBSD, MacOS-X, Windows, OpenBSD, Solaris and Haiku."
 HOMEPAGE="https://bluefish.openoffice.nl/"
-COPYRIGHT="1998-2024 Olivier Sessink and others."
+COPYRIGHT="1998-2025 Olivier Sessink and others."
 LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="https://www.bennewitz.com/bluefish/stable/source/bluefish-$portVersion.tar.gz"
-CHECKSUM_SHA256="97ea1b836a1b30edfc31b63fb4142d7cfd906acf088d61eb4498936313c79f5a"
+CHECKSUM_SHA256="0a518fec2c43db59710dbd591d56eda756b391c9e9b341e2d1e7ba7013bf559d"
+SOURCE_SIG_URI="https://www.bennewitz.com/bluefish/stable/source/bluefish-2.2.17.tar.gz.sig"
+PGPKEYS=(64979277BAFF2D4CB637AC3B291C63A6B78DFBA1)
```

Comment:
Not required atm, was checking (for this): https://github.com/haikuports/haikuporter/pull/305
Need to disable these 2 lines.